tools/docker/docker-compose-mac.yml

```diff
@@ -96,10 +96,12 @@ services:
     security_opt:
       - label=disable
     command:
-      - aap-eda-manage
-      - rqworker
-      - --worker-class
-      - aap_eda.core.tasking.DefaultWorker
+      - /bin/bash
+      - -c
+      - >-
+        aap-eda-manage wait_for_migrations -t ${EDA_WAIT_FOR_MIGRATIONS_TIMEOUT:-30};
```

Comment:
Should we add the same `wait_for_migration` line for [eda-ws](https://github.com/ansible/eda-server/pull/957/files#diff-356c78596b94c6e2943ec4bb62a483fa30a7ac113de4eede37310b0d38cbdfc0L83) in this file?

Reply:
added